### ISSUE TITLE
Finalize tooltip language

### ISSUE THREAD

**Issue by chris-remus** [solana-labs/networkexplorer#566]

**Comment by chris-remus** [comment]
A few questions - 

1 - I'm suggesting we stop using fullnode and only use "validator" or "lead validator" and have asked @mvines for his feedback on that in the spreadsheet.

2 - I think the Uptime measurement is still an open discussion in #425 The tooltip language may change as a result.

3 - I'm not sure how the top validators are ranked, in reference to row 19. Maybe @sunnygleason does?

**Comment by CriesofCarrots** [comment]
> 1 - I'm suggesting we stop using fullnode and only use "validator" or "lead validator" and have asked @mvines for his feedback on that in the spreadsheet.

Yes, definitely. I'm not sold on "lead validator" for anything, though. I think we can just use "leader"
I'll take a pass through the doc with an eye to this.

**Comment by CriesofCarrots** [comment]
I'll also update the "should click through to" column

**Comment by sunnygleason** [comment]
> 3 - I'm not sure how the top validators are ranked, in reference to row 19. Maybe @sunnygleason does?

Right now it's:

`(currentBlock / stageDurationBlocks) + stakeWeightedBonus`
*(expressed as score from 0-100)*

The parameters of the function are totally placeholders - I haven't heard
a final definition from anyone yet. When folks decide the real function &
parameters, I'm happy to implement (or be part of the discussion if
brainstorming is necessary)!

**Comment by CriesofCarrots** [comment]
I've been wondering whether top-validator ranking should have any correlation with the prize metrics @jstarry is developing here: https://github.com/solana-labs/tour-de-sol/tree/master/winner-tool
@mvines , thoughts?

**Comment by mvines** [comment]
The formula @sunnygleason mentioned sounds fine to me.  The final arbiter will be the winner-tool and it uses the entire ledger, whereas this is just a informal snapshot of where everybody ranks throughout the stage.

**Comment by chris-remus** [comment]
There's a TdS leader conversation happening in #485 

In it, I reference the reward program listed [here](https://forums.solana.com/t/tour-de-sol-stage-1-preliminary-compensation-design/79).

Is the tool being developed by @jstarry consistent with that version of the rewards program?

Also, I'd be hesitant to display a leaderboard that's not consistent with the way rewards will be paid out. This can cause much confusion and frustration among validators.

This happened in GoS in a way. While there was no explicit leaderboard, people tried inferring who the leaders were, only to find out at the end, their inferences were inconsistent with the winners, as announced by the Tendermint team.

I'd suggest the leaderboard should be consistent with the winning logic or not shown at all.
